### PR TITLE
[IOTDB-3033] Completing the query and writing interface compared to java

### PR DESCRIPTION
**Pull request by Coding-Deng**
[IOTDB-3033](https://issues.apache.org/jira/browse/IOTDB-3033)
Add a one device string insert interface; 
one device insert aligned string interface;
an interface for querying data by time, 
and an interface for querying the latest time.

Submit unit test script.
Submit a python client interface example.
In python, the query and insert interface is completed.

In the second commit, reformat code by black and flake8

### CODE REVIEW

File: client-py/SessionExample.py [hunk 1]
```diff
@@ -280,6 +280,51 @@
...
+    ["s_01", "s_02", "s_03"],
+    ["s_01", "s_02", "s_03"],
+]
+values_list = [["False", "22", "33"], ["True", "1", "23"], ["False", "15", "26"]]
+session.insert_aligned_string_records_of_one_device(
+    "root.sg_test_01.d_04",
+    time_list,
+    measurements_list,
+    values_list,
+)
```
Comment (HTHou) [inline client-py/SessionExample.py:312]: Move aligned example to SessionAlignedTimeseriesExample.py?

**Review by HTHou — COMMENTED** [review]
Good job, only one thing you may take a look!

File: client-py/SessionExample.py (hunk 1 above)
Comment (Coding-Deng) [reply client-py/SessionExample.py]: Ok, it has been modified as your suggestion

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**